### DISCUSSION TITLE
email_verified_at not filling up after super-admin create a user and user set the password

### DISCUSSION THREAD

**Discussion by fahmiegerton** [chiiya/filament-access-control#60] [Q&A]
As the title said, when a user that has super-admin role create a new user, then the email for resetting password for that user is received, then user set the password, after that they can't login because the email wasn't verified. I don't know if this the package, the filament, or the laravel itself. 

The reason user can't login because the I add this in `User` model
```
public function canAccessPanel(Panel $panel): bool
{
   return $this->hasVerifiedEmail();
}
```

Since it required by [filament docs](https://filamentphp.com/docs/3.x/panels/installation#allowing-users-to-access-a-panel).

Did I miss something?

**Comment by getupkid** [comment]
Any movement on this?

**Comment by chiiya** [comment]
There is no email_verified_at field on the filament_users table. Are you using a custom user model?
